finagle-core/src/test/scala/com/twitter/finagle/util/LoadServiceTest.scala

```diff
@@ -94,6 +95,12 @@ class LoadServiceTest extends FunSuite with MockitoSugar {
     executor.shutdown()
   }
 
+  test("LoadService should ignore packages according to ignoredPaths GlobalFlag") {
```

Comment:
I would like to see a more advanced test.
e.g. ignore "com.twitter.finagle.util" and try to load all the instances of `LoadServiceRandomInterface`, the result Seq should be empty